iphone/Classes/KrollBridge.m

```diff
@@ -770,7 +770,7 @@ -(id)loadCommonJSModule:(NSString*)code withSourceURL:(NSURL *)sourceURL
 	 *      "/Resources/views/login/window.js"  __dirname = "views/login"
 	 */
 
-	NSString *filename = [sourceURL lastPathComponent];
+	NSString *filename = [sourceURL lastPathComponent]; // FIXME This should be the absolute path to the file!
```

Comment:
I opened a separate ticket for parity about this, and didn't really investigate it in this one. Basically __filename should be the full absolute path to the resolved file on all platforms. On iOS and Android we report just the filename (i.e "test.js"). On Windows we report the base name of the file (i.e. "test")

See https://jira.appcelerator.org/browse/TIMOB-23543